_posts/2021-02-24-#21224.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: pr
+date: 2021-02-24
+title: "Halt processing of unrequested transactions"
+pr: 21224
+authors: [ariard]
+components: ["p2p"]
+host: ellemouton
+status: upcoming
+commit:
+---
+
+## Notes
+
+- Currently it is possible for peers to send each other `tx` messages even if the
+  transaction with that txid/wtxid has not been requested.
```

Comment:
It's always possible for a peer to _send_ a transaction, even after this PR. The difference is that we'll then not process that transaction unless it's been requested.